cdap-security/src/main/java/io/cdap/cdap/security/encryption/AeadCipher.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.security.encryption;
+
+import io.cdap.cdap.security.spi.encryption.CipherException;
+import io.cdap.cdap.security.spi.encryption.CipherOperationException;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+/**
+ * An interface intended to represent a service providing AEAD symmetric encryption primitives.
+ */
+public interface AeadCipher {
```

Comment:
It's not a good idea to have two different interfaces with the same name in two different packages. It's a bit confusing to read the code.

Reply:
Done